codespell_lib/data/dictionary.txt

```diff
@@ -4282,6 +4282,9 @@ buuild->build
 buuilds->builds
 bve->be
 bwtween->between
+bypas->bypass
+bypased->bypassed
+bypasing->bypassing
 byteoder->byte order
```

Comment:
Sorry  I know it's not your change, but can this be changed as follows:
```suggestion
byteoder->byteorder, byte order,
```

Otherwise it feels like it's quite risky to code.

Reply:
Sure :+1: